pkg/actors/actors.go

```diff
@@ -598,7 +598,7 @@ func (a *actorsRuntime) callRemoteActor(
 }
 
 func (a *actorsRuntime) isActorLocal(targetActorAddress, hostAddress string, grpcPort int) bool {
-	return strings.Contains(targetActorAddress, "localhost") || strings.Contains(targetActorAddress, "127.0.0.1") ||
+	return strings.Contains(targetActorAddress, "localhost") || strings.Contains(targetActorAddress, a.config.AppChannelAddress) ||
```

Comment:
Any reason why localhost is hardcoded but 127.0.0.1 may not be present? If app channel isn't on localhost shouldn't that be missing too

Reply:
Further investigation has shown this section checks the `daprd` address and not the app channel, so this was unneeded. Changed it back

Reply:
Resolving this now as the code has been reverted